bin/docgen.py

```diff
@@ -72,6 +72,8 @@
 Default configuration
 ---------------------
 
+Default configuration of the UART: {features_params[console_br]}-8-N-1
+
```

Comment:
The console for the Arduino Pro Micro board is over USB and does thus not have any UART configuration. Any baudrate and other serial port settings on the PC will do. We should add another condition in docgen.py to differentiate between UART and USB boards. Something like `value == "CONFIG_START_CONSOLE_UART"` and `value == "CONFIG_START_CONSOLE_USB_CDC"` for `CONFIG_START_CONSOLE`, and write the appropriate console information for each board.

Reply:
I thought about that. But I am not sure how to add an appropriate text to the template (BOARD_RST_FMT). I'd like to leave it separate; text in the template and parameters in the code.

Reply:
The `docgen.py`-script is not very well written in its current version. It should be refactored and cleaned up eventually, but for now I don't really care if you make a sub optimal implementation.